web-dev-server.config.js

```diff
@@ -1,6 +1,26 @@
 import { pfeDevServerConfig } from '@patternfly/pfe-tools/dev-server/config.js';
 
+export const litcssOptions = {
+  include: (/** @type{string[]}*/(/** @type{unknown}*/([
+    /elements\/pf-[\w-]+\/[\w-]+\.css$/,
+    /lib\/.*\.css$/,
+  ]))),
+  exclude: /lightdom/,
```

Comment:
we can probably just add this to the defaults in normalizeOptions. `*-lightdom.css` seems like a reasonably self-explanatory convention to me.